### ISSUE TITLE
Can't see the board for some projects

### ISSUE THREAD

**Issue by willianwrm** [fguillot/kanboard#941]
Sometimes the board seems to be inaccessible, I can see board for projects ID 2 and 3 but for ID 1 I get redirected to the controller app.

If I try "../kanboard/?controller=board&action=show&project_id=**2**" it does work, but if I enter "../kanboard/?controller=board&action=show&project_id=**1**" then I get redirected to "../kanboard/?controller=app&action=index"

Cleaning the cookies (client) seens to do the job, but only for more one time, so I must clean it every time I want see the board for project ID 1. Maybe it's a session problem, some mismatch...

Thanks in advance,
Willian

**Comment by fguillot** [comment]
I don't really understand how that can happen, the action "show" doesn't do any redirection except if you don't have the permission to see to project (forbidden page) or if the project doesn't exists (project not found).

Do you have an idea how I could reproduce the issue? What is the difference between the project 1 and others? Are you administrator?

**Comment by willianwrm** [comment]
It's kind tricky to reproduce the issue, but it's not the first time someone report such thing here.
The project ID 1 is no different from another ones, I did even locked-up directly in database to make sure.

The problem did take place in Firefox (38.0.5), I did try to get the HTTP response using Wireshark but there is no full HTTP GET requests, so my best shot was something wrong with the local cached files; bingo, just did a clean-up and it's working again :)

I will report here in case of buggy again.
There is some screen-shots I got before the cache clean-up:

![01 - sprint is the project 1](https://cloud.githubusercontent.com/assets/7504960/8304741/21bccf3c-197f-11e5-8d3d-c259f67daebf.png)

![02 - if i click it it goes to here](https://cloud.githubusercontent.com/assets/7504960/8304746/28f36180-197f-11e5-8383-eabf7f6c2752.png)

![03 - if i click here note the address bar](https://cloud.githubusercontent.com/assets/7504960/8304747/2d791cb8-197f-11e5-86cb-b6334abb8793.png)

![04 - i do get here again](https://cloud.githubusercontent.com/assets/7504960/8304748/332f2ed6-197f-11e5-94f7-7c2ae41d7c30.png)
